src/components/TodoComponent.tsx

```diff
@@ -0,0 +1,131 @@
+import classNames from 'classnames';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { Todo } from '../types/Todo';
+import { LoaderContext } from './Context/LoaderContext';
+
+interface Props {
+  todo: Todo,
+  onRemoveTodo: (todoId: number) => Promise<void>,
+  onUpdateTodo: (todoId: number, data: Partial<Todo>) => Promise<void>,
+}
+
+export const TodoComponent: React.FC<Props> = React.memo(({
+  todo,
+  onRemoveTodo,
+  onUpdateTodo,
+}) => {
+  const { todoOnLoad, todosOnLoad } = useContext(LoaderContext);
+  const [titleChange, setTitleChange] = useState(todo.title);
+  const todoTitleField = useRef<HTMLInputElement>(null);
+  const [isEditing, setIsEditing] = useState(false);
+
+  useEffect(() => {
+    if (todoTitleField.current) {
+      todoTitleField.current.focus();
+    }
+  }, [isEditing]);
+
+  const todosOnProccessing = (todoId: number) => {
+    if (todoId === 0
+      || todoOnLoad === todoId
+      || todosOnLoad.find(id => id === todoId)) {
+      return true;
+    }
+
+    return false;
+  };
+
```

Comment:
I think, you can create a variable for this, not a function.
```suggestion
  const isTodosOnProccessing = todo.id === 0 || todoOnLoad === todo.id || todosOnLoad.includes(todo.id)
```
It looks understandeble and simple as for me))

Reply:
agree, thank you!